lib/medium-admin.php

```diff
@@ -736,7 +724,7 @@ private static function _get_migration_data() {
   /**
    * Creates a post on Medium.
    */
-  public static function cross_post($post, $medium_post, $medium_user) {
+  public static function cross_post($post, &$medium_post, $medium_user) {
```

Comment:
I wouldn't do this. It would be the only method where we pass by reference, and there's no reason we can't just return the new post.